achyut/src/main/java/com/lftechnology/com/programs/SalutationProgram.java

```diff
@@ -0,0 +1,49 @@
+package com.lftechnology.com.programs;
+
+import java.util.Scanner;
+
+public class SalutationProgram {
+
+    private Scanner fullNameWithSalutationInput;
+    private String fullNameWithSalutation;
+    private String salutation;
+
+    public SalutationProgram() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1186)